src/motor-interface-pru.p

```diff
@@ -28,6 +28,7 @@
 #define CONST_PRUDRAM	   C24
 
 #define QUEUE_ELEMENT_SIZE (SIZE(QueueHeader) + SIZE(TravelParameters))
+#define QUEUE_END (QUEUE_LEN * QUEUE_ELEMENT_SIZE)
```

Comment:
Logically, this would be STATUS_VALUE. It happens to be at the end of the queue, but that is not what logically is there.